runtime/driver.c

```diff
@@ -1178,14 +1178,16 @@ static stz_long bitset_size (stz_long heap_size) {
   return ROUND_UP_TO_WHOLE_PAGES(bitset_size_in_longs << LOG_BYTES_IN_LONG);
 }
 
-#ifdef BUILD_DEBUG_ADAPTER
-int stanza_main (int argc, char* argv[]) {
-  //printf("C: Running program with %d arguments:\n", argc);
-  //for (char** p = argv; *p != NULL; p++)
-  //  printf("  \"%s\"\n", *p);
+//Use 'main' as the standard name for the C main function unless
+//RENAME_STANZA_MAIN is passed as a flag. If it is, then rename 'main'
```

Comment:
Renamed this to a general feature: `RENAME_STANZA_MAIN` to help any other program that embed Stanza.